src/logic/safe/store/middleware/notificationsMiddleware.ts

```diff
@@ -105,7 +107,23 @@ const notificationsMiddleware =
           const safesMap = safesAsMap(state)
           const currentSafe = safesMap.get(safeAddress)
 
-          if (!currentSafe || !isUserAnOwner(currentSafe, userAddress) || awaitingTransactions.length === 0) {
+          const chainId = currentChainId(state)
+          const localStatusedSafeTxHashes = Object.keys(localStatuses(state)?.[chainId] || {})
+
+          if (!localStatusedSafeTxHashes.length) {
+            break
+          }
+
+          const hasLocalStatus = transactions.some((tx) =>
+            localStatusedSafeTxHashes.some((safeTxHash) => tx.id.includes(safeTxHash)),
```

Comment:
Isn't `tx.id` different than `safeTxHash`? Or in this case it's same?

Reply:
Transaction summaries do not return the `safeTxHash`. The `id`, however, contains it.

Reply:
Is it in a random place or can we extract the exact safeTxHash part and use for hash map lookup?

Reply:
From my understanding, it is the last part of the `id` but I am not 100% sure. It would be ideal if the `safeTxHash` was returned as part of the summary but that is something we'd have to discuss with backend in the new year.